static/app/views/issueDetails/groupTags/useGroupTags.tsx

```diff
@@ -66,6 +66,7 @@ export function useGroupTags(
   return useApiQuery<GroupTag[]>(
     makeGroupTagsQueryKey({
       orgSlug: organization.slug,
+      limit: 3,
```

Comment:
can we remove the `limit` from `useGroupTagsReadable` if we are passing it in here?

Reply:
Yup, removed the limit from there (and added limit: 4 to the old UI usage)